app/views/contacts/edit.html.erb

```diff
@@ -20,6 +20,7 @@
       <%= form.govuk_phone_field :phone, width: 10 %>
 
       <%= form.govuk_submit t("contact.edit.save_contact_button") do %>
+        <%= govuk_button_to "Delete", project_contact_path(@project, @contact), method: :delete, warning: true %>
```

Comment:
This doesn't seem to have any warning or confirmation, I think it may rely on some Javascript we don't have.

Would it be safer to have it just open a new view offering a delete button and back button?

Reply:
Yep, `warning: true` is part of UJS which we do not have. 

There might be a wider conversation here about the pattern we use to delete an entity, we either use something quick but JS based or build a more response/request type thing that we can hopefully reuse - it would be worth chatting to @titlescreen about this - I think he has already thought about this and picked the later.

There is a balance to strike here, effort vs value, I would like us to see if we can build a reusable request/response delete but it shouldn't become a time sink - happy to talk this through.

Reply:
I think `warning: true` here is just the red colour of the button.

I didn't actually notice, but the prototype has a delete confirmation screen, I'm just in the middle of doing that now.

Reply:
In the prototype I had put in a delete contact flow where the user would pick change and then on the edit contact page  there would be a delete option at the bottom. This would point them to an are you sure screen. 

I've added it to the contacts [Lucid chart](https://lucid.app/lucidspark/cbfc31ef-8d60-4a81-bdd0-da018d78cbc4/edit?view_items=S-~K8HK3EkYY&invitationId=inv_69b4f3cc-535d-4d7e-9176-16ddc010a0ec#)

![image](https://user-images.githubusercontent.com/13239597/188830571-e2e1af7a-92bd-4ba5-ae90-10ae97c00c7d.png)
![image](https://user-images.githubusercontent.com/13239597/188830613-27e8411c-35cb-47a9-8fb8-eb799e5d15a4.png)

Reply:
@titlescreen Yep, that's what I've followed! Screenshots added to the description.

@jacksonj04 @mec How does that look? not sure about making it reusable

Reply:
That looks like what I would expect! I don't know how reusable we can make it tbh, this might be one of those things like the edit contact screen where it feels like it _should_ be very DRY but actually there's enough nuance in the template etc it's easier to just have each class's own flow.

Reply:
> I think `warning: true` here is just the red colour of the button.

Yep, sorry Joe, I am getting confused between delete conversation and jumping to conclusions!